.github/workflows/ci-model-regression-on-schedule.yml

```diff
@@ -0,0 +1,186 @@
+name: CI - Model Regression 
+
+on:
+  schedule:
+    - cron:  '1 23 * * */7'
+
+env:
+  GKE_ZONE: us-central1-a
+
+jobs:
+  read_test_configuration:
+    name: Reads tests configuration
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    steps:
+    - name: Checkout master
+      uses: actions/checkout@v2
+
+    - name: Download yq
+      run: |
+        curl --location https://github.com/mikefarah/yq/releases/download/3.3.0/yq_linux_amd64 -o yq
+        chmod +x yq
+
+    - run: test -f .github/configs/mr-test.yaml
+
+    - id: set-matrix
+      shell: bash
+      run: echo "::set-output name=matrix::$(./yq -j r .github/configs/mr-test-schedule.yaml)"
+
+  deploy_runner_gpu:
+    name: Deploy Github Runner - GPU
+    needs: read_test_configuration
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Download gomplate
+        run: |-
+          curl -o gomplate -sSL https://github.com/hairyhenderson/gomplate/releases/download/v3.6.0/gomplate_linux-amd64
+          chmod 755 gomplate
+
+      - name: Render deployment template
+        run: |-
+          ./gomplate -f .github/runner/deployment.yaml.tmpl -o runner_deployment.yaml
+
+      # Setup gcloud CLI
+      - uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
```

Comment:
since this is messing with the cluster I'd feel better if we pin an exact code version using a sha instead of using `master` (or fork the repo)

Reply:
Technically, we don't touch the cluster in this step, but I agree. Pin the version out will help to avoid unexpected behavior 👍